spec/features/changing_answers_spec.rb

```diff
@@ -64,7 +64,6 @@
     expect(find("#claim_qts_award_year_on_or_after_cut_off_date").checked?).to eq(true)
 
     choose_qts_year :before_cut_off_date
-    click_on "Continue"
```

Comment:
This was actually clicking on the "Continue session" button in the timeout banner at the top of the page.
This `click_on` is not needed, because the `choose_qts_year` helper does it's own `click_on "Continue"`